stable/hubot/templates/deployment.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "hubot.fullname" . }}
+  namespace: {{ .Values.namespace }}
+  labels:
+    app: {{ template "hubot.name" . }}
+    chart: {{ template "hubot.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ template "hubot.name" . }}
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "hubot.name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      volumes:
+        - name: {{ template "hubot.fullname" . }}-scripts
+          configMap:
+            name: {{ template "hubot.fullname" . }}-scripts
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - name: http
+              containerPort: 8080
+              protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /health
+              port: http
+          readinessProbe:
+            httpGet:
+              path: /health
+              port: http
+          volumeMounts:
+          - name: {{ template "hubot.fullname" . }}-scripts
```

Comment:
Please use a consistent list indentation style throughout the chart. Either use zero or two spaces.